apps/web/src/chat-first/first-run.css

```
@@ -406,11 +406,24 @@ button.cf-first-run__engine-choice:not(.is-selected):hover {
   grid-template-columns: minmax(170px, 1fr) auto;
   gap: 8px;
 }
 
+.cf-first-run__hosted-label-text {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)